src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -532,13 +535,18 @@
         - name: --max-count
           type: int
           short-summary: Maximum nodes count used for autoscaler, when "--enable-cluster-autoscaler" specified. Please specifying the value in the range of [1, 100]
+        - name: --mode
+          type: string
+          short-summary: The mode for a node pool which defines a node pool's primary function. If set as "system", AKS prefers system pods scheduling to node pools with mode `system`. Learn more at https://aka.ms/aks/nodepool/mode.
```

Comment:
System? With capital "S"? Or we ignore case sensitivity?

Reply:
changed to capital "S" for consistency. Also set the parameter type to enum type.